msg_helpers_test.go

```diff
@@ -0,0 +1,104 @@
+package dns
+
+import "testing"
+
+func TestPackDataNsec(t *testing.T) {
```

Comment:
Can you add some comments on how you generated these test cases?

Reply:
added such comment